engine/src/engine.cpp

```diff
@@ -42,6 +42,7 @@ void Engine::initAll(const bool& loadUsbDriver) {
   banner.show(&renderer);
   audio.init();
   pad.init();
+  fontInit(&renderer);

```

Comment:
New font class should be more object oriented (like `font.init(...)`, etc.).
Please refactor it to make it look like the neighboring classes (`pad`, `audio`, etc.)
Please avoid to use `extern` in following hpp file - probably is not needed here.

Reply:
I thought about doing that but I also thought that an exception could be made 😅, since it was something that is used a lot and it would be easier to access the function directly. well now I change it.